tests/main.cpp

```diff
@@ -7,7 +7,7 @@ int main(int argc, char* argv[])
     Catch::Session session;
 
     using namespace Catch::Clara;
-    std::string log_level = "debug", log_file = "stderr";
+    std::string log_level = "trace", log_file = "stderr";
```

Comment:
idk if we want to default this to `trace` since it can easily be specified in the CLI.